src/components/CreateTemplateFromTemplate.tsx

```diff
@@ -33,7 +34,7 @@ export const CreateTemplateFromTemplate = () => {
       "Error creating template from template",
       error || data
     );
-    window.location.href = templatesUrl;
+    <NavigateToExternalUrl to={templatesUrl} />
```

Comment:
why do you use a component declaration here, is no better use a React hook for this behavior?

I see that <NavigateToExternalUrl to={templatesUrl} /> is not render in anytime, this code is just to force the navigation.

Reply:
[_Here_](https://github.com/FromDoppler/doppler-editors-webapp/blob/1f08a114b218bc3d19c1806168917b8828bed1d1/src/components/RequireAuth.tsx#L13-L20) is a similar case but we are return in that case, maybe we can return in this case the `NavigateToExternalUrl`?

Reply:
Oh it seems that I missed the return
```suggestion
    return <NavigateToExternalUrl to={templatesUrl} />
```

Reply:
Is that better?